crate/content.md

```diff
@@ -1,111 +1,40 @@
-
-To start the Crate cluster in containers distributed to three hosts without multicast enabled, run this command on the first node and adapt the container and node names on the two other nodes:
-
-```console
-# HOSTS="crate1.example.com:4300,crate2.example.com:4300,crate3.example.com:4300"
-# HOST="crate1.example.com"
-# docker run -d -p 4200:4200 -p 4300:4300 \
-    --name crate1-container \
-    --volume /mnt/data:/data \
-    --ulimit nofile=65535 \
-    --ulimit memlock=9223372036854775807 \
-        crate \
-        crate \
-          -Des.cluster.name=crate-cluster \
-          -Des.node.name=crate1 \
-          -Des.transport.publish_port=4300 \
-          -Des.network.publish_host="$HOST" \
-          -Des.multicast.enabled=false \
-          -Des.discovery.zen.ping.unicast.hosts="$HOSTS" \
-          -Des.discovery.zen.minimum_master_nodes=2
-```
-
-# The crate Docker Image
-
-To form a cluster from scratch, start a few instances of the Crate container as a background daemon:
-
-```console
-# docker run -d crate crate
-```
-
-To access the admin UI, map port 4200 and point your browser to port tcp/4200 of a node of your choice while you start it or look up its IP later:
-
-```console
-# firefox "http://$(docker inspect --format='{{.NetworkSettings.IPAddress}}' $(docker run -d crate crate)):4200/admin"
-```
-
-For production use it's strongly recommended to use only one container per machine. This will give the best possible performance and by mapping the ports from the Docker container to the host it acts like a native installation. Crate's default ports 4200 (HTTP) and 4300 (Transport protocol).
-
-```console
-# docker run -d -p 4200:4200 -p 4300:4300 crate crate
-```
-
-## Attach Persistent Data Directory
-
-Crate stores all important data in */data*. It's advised to mount this directory to avoid writing within the docker image:
-
-```console
-# docker run -d -v <data-dir>:/data crate crate
-```
-
-## Use Custom Crate Configuration
-
-Starting with 0.55.0, Crate does no longer support providing custom configuration files. However it is still possible to mount Crate's configuration into `/crate/config/crate.yml`.
-
-```console
-# docker run -d -v <custom/config/path>/crate.yml:/crate/config/crate.yml crate crate
-```
-
-For further configuration options refer to the[Configuration](https://crate.io/docs/stable/configuration.html) section of our documentation.
-
-## Environment
-
-Crate recognizes environment variables like `CRATE_HEAP_SIZE` that need to be set with the `--env` option before the actual Crate core starts. As a rule of thumb you may want to [assign about half of your memory ](https://crate.io/docs/reference/en/latest/configuration.html#crate-heap-size) to Crate:
-
-```console
-# docker run -d --env CRATE_HEAP_SIZE=32g crate crate
-```
+# What Is CrateDB?
 
-## Open Files
+[CrateDB](github.com/crate/crate) is a distributed SQL database that makes it
+simple to store and analyze massive amounts of machine data in real-time.
```

Comment:
Doh, sorry for not being more specific -- `./markdownfmt.sh` is just an interface to the particular `markdownfmt` tool we use.  If you use `-d`, it'll show you a "diff" of the formatting changes it expects.  If you use `-w`, it'll write the suggested changes directly to the files for you (which you'll want to review afterwards -- it doesn't always get the intent 100% right).  Leaving the `README.md` completely alone is perfect -- @docker-library-bot will update it (and the Docker Hub description) automatically after this PR merges. :+1: